docs/reference/contributing/target/Watchdog.md

```diff
@@ -1,25 +1,45 @@
-<h3 id="watchdog-port">Watchdog</h3>
+### Watchdog
 
-[Include a brief description here.]
+The Watchdog HAL API provides a low-level interface to the Independent Watchdog Timer of a target. Implementing the Watchdog API is not mandatory, but by implementing it, you can use the hardware watchdog timer to detect and recover from computer malfunctions.
+
+Implement the Reset Reason API when implementing the Watchdog API. The Reset Reason API allows you to detect at boot time whether a watchdog caused the last reset.
 
 #### Assumptions
 
 ##### Defined behavior
 
-[Include any defined behavior in bullet format here.]
+- Sleep and debug modes don't stop the watchdog from counting down.
```

Comment:
Query: Does "watchdog" in this line refer to the API or the watchdog timer or something else?

Reply:
It refers to the hardware watchdog timer that the API manages.